modules/standard/AutoMath.chpl

```diff
@@ -2695,14 +2695,10 @@ module AutoMath {
   }
 
   proc chpl_gcd(in a: int,in b: int): int {
```

Comment:
Maybe this should be a generic helper? (Might have to leave out the abs calls for unsigned types if abs is not already the same as identity for those)